CHANGELOG.md

```diff
@@ -5,32 +5,106 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to the versioning scheme outlined in the [README.md](README.md).
 
-## [Unreleased - Stacks 2.1]
+## [2.1]
 
-This release will contain consensus-breaking changes.
+This is a **consensus-breaking** release that introduces a _lot_ of new
+functionality.  Details on the how and why can be found in [SIP-015](https://github.com/stacksgov/sips/blob/feat/sip-015/sips/sip-015/sip-015-network-upgrade.md),
+[SIP-018](https://github.com/MarvinJanssen/sips/blob/feat/signed-structured-data/sips/sip-018/sip-018-signed-structured-data.md),
+and [SIP-20](https://github.com/obycode/sips/blob/bitwise-ops/sips/sip-020/sip-020-bitwise-ops.md).
+
+The changelog for this release is a high-level summary of these SIPs.
 
 ### Added
 
-- Clarity function `stx-transfer?` now takes a 4th optional argument, which is a memo.
-- Added a new parser which will be used to parse Clarity code beginning with 2.1,
-  resolving several bugs in the old parser and improving performance.
-- Documentation will indicate explicitly which Clarity version introduced each
-  keyword or function. 
-- Clarity2 improvements to traits (see #3251 for details):
+- There is a new `.pox-2` contract for implementing proof-of-transfer.  This PoX
+  contract enables re-stacking while the user's STX are locked, and incrementing
+the amount stacked on top of a locked batch of STX.
+- The Clarity function `stx-account` has been added, which returns the account's
+  locked and unlocked balances.
+- The Clarity functions `principal-destruct` and `principal-construct?`
+  functions have been added, which provide the means to convert between a
+`principal` instance and the `buff`s and `string-ascii`s that constitute it.
+- The Clarity function `get-burn-block-info?` has been extended to support
```

Comment:
This a new function, not an extension of an existing one.

Reply:
Fixed